gopem/helper.py

```diff
@@ -116,7 +116,7 @@
 StyleList = sorted(StyleTable)
 StyleList.insert(0, StyleList.pop(StyleList.index("Solid")))
 
-ScaleList = ["Linear", "Log"]
+ScaleList = ["linear", "log"]
```

Comment:
It would be better to use `self.x_scale.currentText().lower()` and `self.y_scale.currentText().lower()` within the `mainwindow/axis_changed` method instead.

Reply:
I tested it on an earlier version of Matplotlib, and it worked perfectly! 💯

**OS**: Windows 10

**Python**: 3.5.2

**matplotlib**: 3.0.3

**opem**: 1.4

Reply:
thanks for testing, I'd add the change and keep these new `ScaleList`